generator/src/main/java/com/scottlogic/deg/generator/GenerateExecute.java

```diff
@@ -64,4 +79,10 @@ public void run() {
             e.printStackTrace();
         }
     }
+
+    private static boolean validationResultShouldHaltExecution(Collection<ValidationAlert> alerts) {
```

Comment:
This wouldn't be needed if ValidationResult was used instead

Reply:
This isn't the case as `ValidationResult` holds a collection of strings, it does not retain their criticality. `ValidationAlert` includes the criticality for the message. The process should only exit if one or more `ERROR` messages are returned